apps/condo/domains/ticket/hooks/useTicketDocumentGenerationTask.tsx

```diff
@@ -52,8 +53,13 @@ export const useTicketDocumentGenerationTask = ({ ticket, user, timeZone }) => {
                 key: 'generate-document-of-completion-works',
                 onClick: getHandleClick(TICKET_DOCUMENT_TYPE.COMPLETION_WORKS),
             },
+            ticket.isPaid && get(invoices, '0') && isSupportedDocumentTypeByLocale(TICKET_DOCUMENT_TYPE.PAID_WORKS, locale) && {
+                label: PaidWorksLabel,
+                key: 'generate-document-of-paid-works',
```

Comment:
I'd have vent with

document-of-works-completion
document-of-paid-works-completion

Reply:
generate-document-of-works-completion
generate-document-of-paid-works-completion